source/android/adaptivecards/build.gradle

```diff
@@ -36,6 +37,30 @@ signing {
     sign configurations.archives
 }
 
+task sourceJar(type: Jar) {
+    from android.sourceSets.main.java.srcDirs
+    classifier "sources"
+}
+
+publishing {
+    publications {
+        adaptivecards(MavenPublication) {
+            groupId 'io.adaptivecards'
+            version '1.0.1'
```

Comment:
I'm not sure about the versioning, and left as hardcoded s tring. how should we version it?

Reply:
Ideally this should take the nuget version from the custom.props and apply it.

---
In reply to: [215369799](https://github.com/Microsoft/AdaptiveCards/pull/1853#discussion_r215369799) [](ancestors = 215369799)

Reply:
thansks for the good suggestion; will do

---
In reply to: [215424137](https://github.com/Microsoft/AdaptiveCards/pull/1853#discussion_r215424137) [](ancestors = 215424137,215369799)

Reply:
@gillesk@microsoft.com  changed the version to read from custom.props and concat file date time stamp string to version.

---
In reply to: [215424137](https://github.com/Microsoft/AdaptiveCards/pull/1853#discussion_r215424137) [](ancestors = 215424137,215369799)